packages/components/src/components/hds/text/body.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+import {
+  HdsTextSizeValues,
+  HdsTextWeightValues,
+  type HdsTextColors,
+  type HdsTextTags,
+  type HdsTextAligns,
+  type HdsTextWeights,
+} from './types.ts';
```

Comment:
I suggest we separate the imports for [reasons](https://github.com/embroider-build/addon-blueprint/issues/272#issuecomment-1992512789). Similarly for the other files (`code.ts`, `display.ts`, `index.ts`).
```suggestion
import { HdsTextSizeValues, HdsTextWeightValues } from './types.ts'
import type { HdsTextAligns, HdsTextColors, HdsTextTags, HdsTextWeights } from './types.ts';
```